exporter/datadogexporter/factory.go

```diff
@@ -59,6 +59,8 @@ func createDefaultConfig() configmodels.Exporter {
 			TCPAddr: confignet.TCPAddr{
 				Endpoint: "", // set during config sanitization
 			},
+			SendMonotonic: true,
```

Comment:
This is technically a breaking change. Should we add a warning and wait for one release until we actually enable this?

Reply:
Up to you

Reply:
We discussed this internally at Datadog and want to enable this from the start.